/user_data/vayzenbe/GitHub_Repos/vonenet/vonenet/params.py:59: RuntimeWarning: invalid value encountered in true_divide
  ny_dist_marg = n_joint_dist / n_joint_dist.sum(axis=1, keepdims=True)
/home/vayzenbe/anaconda3/envs/ml/lib/python3.9/site-packages/torch/functional.py:478: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at  /opt/conda/conda-bld/pytorch_1659484775609/work/aten/src/ATen/native/TensorShape.cpp:2895.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
libraries loaded...
Neuronal distributions gabor parameters
Model:  VOneCORnet-S
Traceback (most recent call last):
  File "/user_data/vayzenbe/GitHub_Repos/kornet/modelling/extract_acts.py", line 104, in <module>
    model, transform, layer_call = load_model(model_arch)
  File "/user_data/vayzenbe/GitHub_Repos/kornet/modelling/model_loader.py", line 141, in load_model
    checkpoint = torch.load(f'{weights_dir}/{model_arch}_best_1.pth.tar')
  File "/home/vayzenbe/anaconda3/envs/ml/lib/python3.9/site-packages/torch/serialization.py", line 699, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/vayzenbe/anaconda3/envs/ml/lib/python3.9/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/vayzenbe/anaconda3/envs/ml/lib/python3.9/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: '/lab_data/behrmannlab/vlad/kornet/modelling/weights/modelling/weights/vonenet_r_stylized-ecoset_best_1.pth.tar'
